lib/bluetooth_low_energy/services/mqtt_ble/aws_iot_mqtt_ble.c

```diff
@@ -634,11 +671,23 @@ void vRXLargeMesgCharCallback( BLEAttribute_t * pxAttribute,
 
         			if( pxWriteParam->xLength < mqttBLETRANSFER_LEN( usBLEConnMTU ) )
         			{
+                                        for ( x =0; x < pxService->xConnection.xRecvOffset; x++ )
+                                        {
+                                            if( pxService->xConnection.pRecvBuffer[x] == '\\' )
+                                            {
+                                                pBuf = ( pxService->xConnection.pRecvBuffer ) + x - 100;
+
```

Comment:
it is a magic number. Maybe do a #define

Reply:
Its fixed in second commit.